WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -104,8 +104,8 @@ + (NSArray *)routes
   if (elementResponseAttributes) {
     [FBConfiguration setElementResponseAttributes:elementResponseAttributes];
   }
-  if (capabilities[FB_CAP_MAX_TYPING_FREQUENCY]) {
-    [FBConfiguration setMaxTypingFrequency:[capabilities[FB_CAP_MAX_TYPING_FREQUENCY] unsignedIntegerValue]];
+  if (capabilities[FB_SETTING_MAX_TYPING_FREQUENCY]) {
```

Comment:
see below

Reply:
Thank you. Fixed in https://github.com/appium/WebDriverAgent/commit/75d26ca34ccdee7fdfbb5318d346cbf0a9a822b0